### ISSUE TITLE
Rename Circle?

### ISSUE THREAD

**Issue by bernardmariechiquet** [holacracyone/Holacracy-Constitution#243]
Tension :
• Many people take a long time to understand the logic of the structure, i.e. holarchy
• Many people have a hard time going from "I work for myself and my boss" to "I work for my role, my circle and the purpose of the organization". They find it hard to believe that their new boss is the Purpose of the entire company.
I propose to replace the word Circle by the word Holon. The reason comes from the fact that I have observed the notion of hierarchically-based separation surviving in Holacracy when it relies on a holarchy without saying so and without taking advantage of these positive effects - namely a mental model that helps to understand that the role (the cell) is also the circle (the organ) and also the anchoring circle (the whole body).
It is difficult even after years to fully understand Holacracy's holarchy and its functioning with membranes as long as one does not change one's mental mind by leaving the separation one and linking the proper functioning of the role to the other. that of the circle (term too flat) and that of the entire organization.
To bring directly this metaphor of the human body (cell, organ) through the language used in the constitution will undoubtedly facilitate and lead to a better understanding of what the holarchic structure is.

**Comment by LouisChiquet** [comment]
Totally rational and easier to explain!

**Comment by tylerdanke** [comment]
The word Holons is not one that i believe that English speakers understand
naturally. I believe that it is important to use words that are used
commonly whenever possible.

**Comment by bernardmariechiquet** [comment]
@tylerdanke I fully agree... and it's worth getting into it at the very beginning IMO as it's required with mature practice
And I also think it is easy to get it using the metaphor of the body - I've tried it with 2 clients from the very beginning after adoption and they get it.
I do think we should dare using it at the root into the constitution instead of continuing thinking people don't know it. They will have less hard time to get it.

**Comment by brianjrobertson** [comment]
@bernardmariechiquet This change doesn't make sense to me; it adds complexity and non-natural language, and loses the ability for people to overly-simplify at the beginning, which I think is more beneficial than harmful.  I'll leave this issue open in case you or others can weigh in to convince me otherwise; I'm open but skeptical of this one.

**Comment by bernardmariechiquet** [comment]
I understand that holon is not a natural language at least for now, so I suggest to rename Circle as Organ. An organ is very natural language and also very natural to what is Holacracy and holarchy. Definitively Circle is making Holacracy more difficult to get and to embrace. A circle is a word both used in Maths and in Collective Intelligence like sociocracy. For the latter, it creates confusion between the team and what is a circle, a part, a holon, an organ of the Org. 
With organ at play, people will still have the ability to over-simplify but in at least with the appropriate mental model as the cell/body metaphor is even more natural than the abstract word *Circle*. So there is more benefit and no harm.
I wonder if part of the skepticism some may have here is due to the fact that the word Circle has been used for some years now, and it may be difficult to change it because habits is already in place.

**Comment by bernardmariechiquet** [comment]
And coming back to holon. Holon may not be natural language but it is the right concept whereas Circle is not at all. IMO It's better to overcome a difficulty initially than to go in the wrong direction using a word that is a false friend and adopting bad habits. The risk is to increase the bar to overcome the PowerShift Dip by not moving to the appropriate mental model.
And I would challenge that Holon may be more difficult to get initially - for many people a circle is not natural language too, except if you are used to collective intelligence processes which by the way creates, by the way, a new trap and so much confusion. Most of the people I encounter in organizations are new to both words - circle and holon.  They have only the maths figure in mind which does not help.
But the advantage with holon is that we've got a powerful and very familiar metaphor.

**Comment by brianjrobertson** [comment]
Hmm, I do understand the value you're pointing to @bernardmariechiquet, though I believe there is more value (much more) in keeping the constitution as close to "jargon free" as possible, using only easy-to-relate-to words.  So much so in fact that I think it'd be better to call it "team" over "holon" or "organ", despite the obvious drawbacks.  The value gained by simple and relatable language seems far, far more important to me than the in-practice drawback you describe (and I've experienced directly many times as well).  So, to consider this further, I'm going to need to hear from many other coaches who think I'm wrong here, or a **very** compelling argument (not just for why holon would be useful - I get that - but for why that's actually more important than the value of keeping simple and relatable language).

**Comment by jeanlucchristin** [comment]
I like « team » 
it’s much better than circle for all the reasons BMChhiquet mentioned. 

team is super simple et common langage

**Comment by bernardmariechiquet** [comment]
@brianjrobertson I understand your argument and the value of keeping the constitution as close to "jargon free" as possible.  And this is a challenge, may be irrealistic. All the words like Role, or Circle or Accountability or Policy or Purpose aren't "jargon free" - in fact, they are new jargon to understand anyway. Team would be a terrible option IMO as this concept has very little chance to get different meanings. At least with "circle" the practitioner add another meaning to the usual word "circle".
From a user perspective (I mean a practitioner) I think there is huge value in finding the right words, accepting the minimum sufficient jargon to get to the right mental model easier and quicker. Two polarities here. My sense is there should be a time when words like circle should be changed in order to get the powershift tipping point with companies. I think this is the time to do so and may be I'm wrong.

**Comment by tylerdanke** [comment]
Team sounds fantastic!

**Comment by jeanlucchristin** [comment]
@bernardmariechiquet I did not observe that words like, Accountability or Purpose are not "jargon free"

  the companies in which I worked have been using the purpose and accountability for decades in their jobs description. 

They are used to these words.

**Comment by LouisChiquet** [comment]
Calling it team is a huge danger, as a Circle is everything else than a Team. And as it's already difficult to make people understand the meaning of the Circle, re-defining the meaning of a word they've been used to use (at least in French language) A LOT would make things way more harder to change the mindset.

Accountability, domain, purpose and circle are perfectly NOT jargon-free in the 30s companies I intervened in (mostly French), with the exception of maybe 3 for purpose and circle (where they kinda already had the word in use). Using “easy-related words” would be also a massive danger, as there are no changes if you don't change the words (look at the French Revolution, Industrial Revolution, Digital Revolution, tons of new words and ideas appeared at those times, creating new words or reabiliting olders which weren't much used, and as we say in French, “_words are the bodies of thoughts_”), and the fact of using new words in this new change helps to change the mindset, as they will stick a new definition to a word they never or generally didn't hear a lot, like Holon (while Circle it's the geometric figure like Bernard Marie said, and it is heavily associated with collective intelligence, in addition, it limits the Holarchy to a 2D figure, which it isn't).

**Comment by bernardmariechiquet** [comment]
@jeanlucchristin In my experience, it takes months for people to get what is really an accountability as defined in the constitution.

**Comment by bernardmariechiquet** [comment]
@LouisChiquet Love your argument re new words or rehabilitating olds unused ones during at revolutions - it resonates a lot for me, thanks.

**Comment by bernardmariechiquet** [comment]
What about "Cell"?

**Comment by tylerdanke** [comment]
Jail cell?

Tyler Danke
purelypoultry.com/ | 920-732-6537 | tyler@purelypoultry.com

**Comment by bernardmariechiquet** [comment]
Biological Cell

**Comment by gmitterer** [comment]
@bernardmariechiquet understand where you are coming from especially supporting the distinctions by explicit wording. On the other hand i heard many complaints about complicated wording as well from clients who felt it was creating more resistance than it supported a mindshift. My rule of thumb would be simplicity over uniqueness. The concepts have to be explained  and (re-) learned anyhow. Having said that i have some clients that started to call circles "cells" and it was pretty helpful for them. So if any change i would opt for cell.

**Comment by bernardmariechiquet** [comment]
Thanks @gmitterer!
From my experience, it's even more difficult to explain the concept with an inappropriate word than with the right one from start. In any case, we'll hear many complaints with both options.

**Comment by julianeroell** [comment]
In German-speaking contexts (and with Art of Hosting people ;-)) "Circle" is a well-established concept, easy to grasp and clearly differentiated from "team". Holacracy Circles are somewhat different from what one ordinarily understands as a "Circle", but I agree with Brian when he says that _"the ability for people to overly-simplify at the beginning (...) is more beneficial than harmful."_ 

I don't like "Holon" at all (it's not a commonly understood word, and a difficult concept to grasp), and I dislike the biological metaphors: I think that as long as we are describing a _rule-based system of governance_, clear technical terms are helpful.

When describing _the practise that builds on the rules_, we may use other terms, but these, I think, will differ from organisation to organisation. In some places, "Circles" are "teams", in others they are not.

TLDR: "Circle" works very well here, does very little harm, and I see no better term available at the moment or any benefit from changing to one of the suggested terms.

**Comment by bernardmariechiquet** [comment]
Interesting perspective @martinaroell - thank you all for your feedback - I'm still sensing a tension on this as I collect more and more data from the field when coaching clients on a daily practice, I observe how such metaphor (the body one) helps people get what is an holarchy and abandon the old mental model that induced separation to move forward to a new one. 
I 'll continue getting more data on this...

**Comment by cysnake4713** [comment]
As a Non-English speakers， holon or cell is much harder to understand, At first contact of holacracy, circle quickly give me the image of the holacracy organization should be, it's very vivid.

**Comment by brianjrobertson** [comment]
I'm closing this issue in light of the feedback, and my own sense of the high importance of simple/accessible language for new practitioners.

**Comment by LouisChiquet** [comment]
Circle been used during decades by many methods, and Holacracy bring a totally different sense, along a 3D sense (not a 2D no one which is intrinsic to this geometry figure). During French Revolution, Industrial Revolution, and even if you look today, Digital Revolution, new words are created (just take a look at Internet, Website, Darkweb or even Holacracy which is a new word created, an average of 150 new words annually enters into the French dictionary), or some others  being reused like Holarchy or Republic.
To help: http://www.victorianweb.org/history/language/courtney1.html
Also I don't understand the general feeling, of disliking the term, let's call a dog a dog. Holacracy have Holarchy in it, Holarchy coming itself from Holon. Don't expect change if you don't change words.

Holon terms give a major change to how we introduce Holacracy, as effectively, when you introduce Holacracy to production/manufacturing/sales/stores or any other sector, from my experience, you always have this path of not understanding what a role mean, and saying “it's not in my role, I don't help”, while what would make sense, and would their common sense ask them to do? Help naturally. And the term Holon, give the philosophy of, yes, you're a cell of the organ, but if another cell of the organ is going bad, you can be pretty sure it'll impact you and you want the organ and body in general to be good, not just your cell, otherwise it's useless, and the fact of switching from Circle to Holon would make this philosophy intrinsic to Holacracy.
